.github/workflows/cypress.yml

```diff
@@ -0,0 +1,10 @@
+name: Run Cypress
+on: [pull_request]
```

Comment:
will this also reset / trigger again if new content is pushed to a pr?

Reply:
see pr description